models/OpenAPI/Parser.cfc

```diff
@@ -219,6 +219,12 @@ component name="OpenAPIParser" accessors="true" {
         };
         objects.each( function( item, index ) {
             if ( isStruct( item ) ) {
+			
+				// If `item` is an instance of Parser, we need to flattin it to a CFML struct
+                if ( isInstanceOf( item, "Parser" ) ) {
```

Comment:
Unfortunately, `isInstanceOf` is extremely non-performant.   Change this use `findNoCase( "Parser", getMetaData( item ).name )` here instead to speed it up.

Reply:
> Unfortunately, `isInstanceOf` is extremely non-performant. Change this use `findNoCase( "Parser", getMetaData( item ).name )` here instead to speed it up.

No problem.  That's a neat trick! Thanks for letting me know. I'll take care of it.